apis/python/src/tiledbsoma/io.py

```diff
@@ -8,7 +8,7 @@
 import pandas as pd
 import pyarrow as pa
 import scipy.sparse as sp
-import tiledb
+import somacore
```

Comment:
maybe `from somacore import options` or `from somacore.options import PlatformConfig` so you can use `options.PlatformConfig` or just `PlatformConfig` down the line

Reply:
done!